src/main/java/Main.java

```diff
@@ -15,7 +15,7 @@ public static void main(String[] args) {
         chromeDriver.findElement(By.xpath("//input[@id='advancedsearchmainfield']")).sendKeys("QA");
         chromeDriver.findElement(By.xpath("//input[@id='advancedsearchmainfield']")).sendKeys(Keys.RETURN);
         List<WebElement> vacancy = chromeDriver.findElements(By.xpath("//div[@class='vacancy-serp']"));
-        //Косячный xPath на 17 строчке
+        //Косячный xPath на 17 строчке)))))))))
```

Comment:
Try to find every item, instead of getting one div, which is wrapper (handler) of every internal item.
"//div[@class='vacancy-serp-item ']"  will be better